### PR TITLE
Docs edit initial

### PR DESCRIPTION
**Pull request by vsnyc**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/partner_editable/overview_target_and_usage.adoc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Replace the content in <>
+// Identify your target audience and explain how/why they would use this Quick Start.
+//Avoid borrowing text from third-party websites (copying text from AWS service documentation is fine). Also, avoid marketing-speak, focusing instead on the technical aspect.
+
+This Quick Start reference deployment guide provides step-by-step instructions for deploying {partner-product-short-name} on the AWS Cloud.
```
Comment (sshvans) [inline docs/partner_editable/overview_target_and_usage.adoc:5]:
```suggestion
This Quick Start reference deployment guide provides step-by-step instructions for deploying {partner-product-short-name} server on the AWS Cloud.
```

File: docs/partner_editable/licenses.adoc [hunk 2]
```diff
@@ -0,0 +1,11 @@
+// Include details about the license and how they can sign up. If no license is required, clarify that. 
+
+This Quick Start builds the IBM MQ environment by using a pre-built Amazon Machine Image (AMI) with IBM MQ installed on the Ubuntu operating system.
+
+The Quick Start requires a license for IBM MQ. To use the Quick Start in your production environment, sign up for an IBM MQ program license at https://www.ibm.com/account/reg/us-en/subscribe?formid=urx-17867[IBM developerWorks] (requires an IBMid). You’ll need to place the license key in an Amazon Simple Storage Service (Amazon S3) bucket and specify its location when you launch the Quick Start. (For details, see link:#step-2.-sign-up-for-an-ibm-mq-license[step 2] of the deployment instructions.)
```
Comment (sshvans) [inline docs/partner_editable/licenses.adoc:5]: For some reason "Step 2" cross-reference link didn't work for me. Please double check it.

File: docs/partner_editable/licenses.adoc [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+If you don’t have a license file, the Quick Start will deploy an IBM MQ trial license, which allows 90 days of free usage in a non-production environment. After this time, the queue manager will expire. You can purchase support at an additional cost, or use the forums.
+
+If the license file is in place at stack creation time, the IBM MQ server will automatically use the production license. To upgrade from a trial license to a production license after deployment, follow the instructions in the https://www.ibm.com/support/knowledgecenter/SSFKSJ_9.0.0/com.ibm.mq.ins.doc/q008943_.htm[IBM Knowledge Center].
```
Comment (sshvans) [inline docs/partner_editable/licenses.adoc:9]: External links should open in new tab. Please check all the external links.
```suggestion
If the license file is in place at stack creation time, the IBM MQ server will automatically use the production license. To upgrade from a trial license to a production license after deployment, follow the instructions in the https://www.ibm.com/support/knowledgecenter/SSFKSJ_9.0.0/com.ibm.mq.ins.doc/q008943_.htm[IBM Knowledge Center^].
```

File: docs/partner_editable/additional_info.adoc [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+[start=3]
+. In the IBM MQ Console login screen, enter the user name and password you created during deployment. (The default user name is *mqconsoleadmin*.)
+
+:xrefstyle: short
+[#login_screen1]
+.{partner-product-short-name} Console login screen
+[link=images/login_screen.png]
+image::../images/login_screen.png[image,width=648,height=439]
+
+The Console initially shows the default queue manager. If the status of the queue manager *QM1* (or the name you entered for the *Queue Manager Name* parameter in step 3) is *Running*, as shown in Figure 5, then you have successfully deployed your queue manager.
```
Comment (sshvans) [inline docs/partner_editable/additional_info.adoc:38]: You might want to add link to the diagram for "Figure 5" text in this para.

File: docs/partner_editable/additional_info.adoc [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+:xrefstyle: short
+[#queue_manager1]
+.Displaying the local queue manager
+[link=images/queue_manager.png]
+image::../images/queue_manager.png[image,width=648,height=439]
+
+You can use the IBM MQ Console to perform administration tasks such as stopping and starting queue managers and creating objects such as queues and channels. In the following steps, you’ll use the console to verify that the server is working correctly and add a message to a queue.
+
+. In the console window, choose *Add widget*.
+
```
Comment (sshvans) [inline docs/partner_editable/additional_info.adoc:49]: The indentation of the images are little off. The image should be at the same indentation as the bullet point, however, that's not how its rendered. Asciidoc doesn't do it by default. To do that, you need to use "List continuation" identifier which is a plus(+) symbol. Here is a [documentation](https://asciidoctor.org/docs/asciidoc-writers-guide/#list-continuation) of it
```suggestion
+
```

File: docs/partner_editable/additional_info.adoc [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+:xrefstyle: short
+[#add_widget1]
+.Adding a widget
+[link=images/add_widget.png]
+image::../images/add_widget.png[image,width=158,height=50]
+
+[start=2]
+. In the *Add a new widget* dialog box, choose *Queues* to display the predefined queue, *Q1*.
+
```
Comment (sshvans) [inline docs/partner_editable/additional_info.adoc:58]:
```suggestion
+
```

File: docs/partner_editable/additional_info.adoc [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+:xrefstyle: short
+[#add_widget2]
+.Creating a queues widget
+[link=images/create-queue-widget.png]
+image::../images/create-queue-widget.png[image,width=634,height=500]
+
```
Comment (sshvans) [inline docs/partner_editable/additional_info.adoc:64]:
```suggestion
+
```

File: docs/partner_editable/additional_info.adoc [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+:xrefstyle: short
+[#view-queue1]
+.Viewing the predefined queue
+[link=images/view-queue.png]
+image::../images/view-queue.png[image,width=641,height=246]
+
+[start=3]
+. On the toolbar, choose the envelope icon to put a test message on the queue.
+
```
Comment (sshvans) [inline docs/partner_editable/additional_info.adoc:73]:
```suggestion
+
```

File: docs/partner_editable/additional_info.adoc [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+:xrefstyle: short
+[#env-icon1]
+.Envelope icon
+[link=images/envelope-icon.png]
+image::../images/envelope-icon.png[image,width=222,height=68]
+
+Figure 9: Envelope icon
```
Comment (sshvans) [inline docs/partner_editable/additional_info.adoc:80]: I think this is not needed.
```suggestion

```

File: docs/partner_editable/additional_info.adoc [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+
```
Comment (sshvans) [inline docs/partner_editable/additional_info.adoc:81]:
```suggestion
+
```

File: docs/partner_editable/additional_info.adoc [hunk 11]
```diff
@@ -0,0 +1,113 @@
...
+:xrefstyle: short
+[#place-message1]
+.Placing a message in the queue
+[link=images/place-message.png]
+image::../images/place-message.png[put_message,width=620,height=226]
+
+[start=4]
+. On the *Queues on QM1* window toolbar, you can now choose the folder icon to browse the messages on the queue.
+
```
Comment (sshvans) [inline docs/partner_editable/additional_info.adoc:90]:
```suggestion
+
```

File: docs/partner_editable/additional_info.adoc [hunk 12]
```diff
@@ -0,0 +1,113 @@
...
+:xrefstyle: short
+[#dir-icon1]
+.Folder icon
+[link=images/folder-icon.png]
+image::../images/folder-icon.png[image,width=222,height=68]
+
```
Comment (sshvans) [inline docs/partner_editable/additional_info.adoc:96]:
```suggestion
+
```

**Review by sshvans — CHANGES_REQUESTED** [review]
Overall it looks good, with few minor suggestions inline.
In general, I would suggest to look for the following:
 * Make external links open in a new tab by adding ^ at the end of the link text.
Additionally, "Security" section is missing in the doc. Not sure if it it was purposely left or a miss.

File: docs/partner_editable/licenses.adoc (hunk 3 above)
Comment (vsnyc) [reply docs/partner_editable/licenses.adoc:9]: Accepted and I've updated all the external links.

File: docs/partner_editable/additional_info.adoc (hunk 4 above)
Comment (vsnyc) [reply docs/partner_editable/additional_info.adoc:38]: Updated

File: docs/partner_editable/licenses.adoc (hunk 2 above)
Comment (vsnyc) [reply docs/partner_editable/licenses.adoc:5]: Updated - this was due to word to asciidoc conversion in pandoc. It is now `link:#_sign_up_for_an_ibm_mq_license[step 2] `

File: docs/partner_editable/additional_info.adoc (hunk 4 above)
Comment (vsnyc) [reply docs/partner_editable/additional_info.adoc:38]: Updated to use a reference

**Comment by vsnyc** [comment]
> Overall it looks good, with few minor suggestions inline.
> In general, I would suggest to look for the following:
> 
> * Make external links open in a new tab by adding ^ at the end of the link text.
> 
Done.

> Additionally, "Security" section is missing in the doc. Not sure if it it was purposely left or a miss.

Yes, this is on purpose. The Security section is from a very old template we used to have. It is no longer a part of the boilerplate and I didn't see a lot of value in keeping it.

I've incorporated all the other suggestions in commit: https://github.com/aws-quickstart/quickstart-ibm-mq/pull/22/commits/49d7c4b4aa1d059fd8e522024b1655f6edccd392

**Review by sshvans — APPROVED** [review]

**Pull request merged**